### DISCUSSION TITLE
message not visible at browser

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#367] [Q&A]
hi,
thanks for your valuable inputs. finally i have install and configure nginx & Inbucket & can access the domain name. but when i try to push mail in mailbox. i cant see mails on browser.  i have follow your config file n did same but not able to see mails. kindly guide where i am missing plz.

**Comment by jhillyerd** [comment]
Can you be more specific about what you do and do not see?  ie does Inbucket web UI load but there are no messages?

If you want to make sure that Inbucket is able to receive & store mail, you can use telnet or netcat to talk to its SMTP port directly:  https://www.linuxjournal.com/content/sending-email-netcat

If Inbucket is able to receive mail, then you'd need to diagnose other portions of your configuration (ie DNS, sending hosts/applicatoins)

**Comment by techsachin1** [comment]
hi
really thanks for your support. I have install my Inbucket on cloud. i am
able to see inbucket web page when i enter my domain name.i.e. example.xyz.
i can reach to that inbucket page in browser. but when i send mail to
***@***.*** i am not able to see any mail on web browser. if i type
***@***.*** in mailbox portaion on inbucket page but not able to see
coming mails.

This is my en variable. do i need to change anything here
Environment=INBUCKET_LOGLEVEL=warn
Environment=INBUCKET_SMTP_ADDR=0.0.0.0:25
Environment=INBUCKET_POP3_ADDR=0.0.0.0:1100
Environment=INBUCKET_WEB_ADDR=0.0.0.0:9000
Environment=INBUCKET_WEB_UIDIR=/usr/local/share/inbucket/ui
Environment=INBUCKET_WEB_GREETINGFILE=/etc/inbucket/greeting.html
Environment=INBUCKET_STORAGE_TYPE=file
Environment=INBUCKET_STORAGE_PARAMS=path:/var/local/inbucket
***@***.***

# Uncomment line below to use low numbered ports
#ExecStartPre=/sbin/setcap 'cap_net_bind_service=+ep'
/usr/local/bin/inbucket

Environment=INBUCKET_SMTP_DOMAIN=example.xyz
Environment=INBUCKET_SMTP_MAXRECIPIENTS=2000
#Environment=INBUCKET_POP3_DOMAIN=example.xyz
Environment=INBUCKET_STORAGE_RETENTIONPERIOD=48h

**Comment by techsachin1** [comment]
when i run curl http://localhost:9000 its showing  You need to enable
JavaScript to run this app. is this mandetory

**Comment by jhillyerd** [comment]
The result you see in curl is normal, as most of the app loads via javascript.  Main thing to test that your nginx config is working would be to load the monitor tab and make sure the italic "Connected." text appears, compare to http://demo.inbucket.org/monitor -- if not, then websocket proxy isn't working.

Please try to use netcat or telnet to send an email (see linked article), first try from the machine inbucket is running on to make sure inbucket can store the email.  Then try externally to verify networking.

**Comment by techsachin1** [comment]
Hi,
All mail received thanks for your support. i have one question, when i
received mail on webpage. the hyperlink or e.g. verification link in mail
is clickable from browser. what changes i have to do.

**Reply by jhillyerd** [reply]
Try toggling between the "Plain Text" and "Safe HTML" tabs when viewing.  The text mode will make text URLs clickable, HTML won't unless they are a proper `<a>`
